### PR TITLE
Update Unit Test coverage by 10%

### PR DESCRIPTION
**Pull request by drosenberg123**
- added tests to improve coverage by 25% in all 4 areas
- hit 85% critical coverage (internal) goal
- minor clean-up of older code (some tiny bugs surfaced by unit tests), included prettier fixes

### CODE REVIEW

File: src/RoleObjects/AccessibilityObject.js [hunk 1]
```diff
@@ -484,12 +484,20 @@ export default class AccessibilityObject {
...
+      this._displayObject.hasEventListener('click') ||
+      this._displayObject.hasEventListener('mousedown') ||
+      this._displayObject.hasEventListener('pressup');
+    const five = one ? two : three && four;
+    // console.warn('1', one);
+    // console.warn('2', two);
+    // console.warn('3', three)
+    // console.warn('4', four)
+    // console.warn('5', five)
+    return five;
```
Comment (drosenberg123) [inline src/RoleObjects/AccessibilityObject.js:500]: - testing

File: src/RoleObjects/AccessibilityObject.js [hunk 2]
```diff
@@ -875,9 +883,9 @@ export default class AccessibilityObject {
     Object.defineProperty(event, 'keyCode', {
       get() {
         // eslint-disable-next-line no-console
-        console.warn(
-          '"keyCode" Property is being deprecated, and will be removed in future major version of the createjs-accessibility module. Please use "key" property instead.'
-        );
+        // console.warn(
+        //   '"keyCode" Property is being deprecated, and will be removed in future major version of the createjs-accessibility module. Please use "key" property instead.'
+        // );
```
Comment (drosenberg123) [inline src/RoleObjects/AccessibilityObject.js:888]: - testing

File: src/RoleObjects/MultiLineTextBoxData.js [hunk 3]
```diff
@@ -283,7 +283,9 @@ export default class MultiLineTextBoxData extends AccessibilityObject {
    * @returns {String} string value for the text input field
    */
   get value() {
-    return this._reactProps.value;
+    const elementId = this._reactProps.id;
+    const element = document.getElementById(elementId);
+    return element.value;
```
Comment (drosenberg123) [inline src/RoleObjects/MultiLineTextBoxData.js:288]: - this is a fix... `value` prop was never set (it was changed to store on the DOM element itself in a previous PR)

File: src/RoleObjects/SingleSelectListBoxData.test.js [hunk 4]
```diff
@@ -100,7 +102,7 @@ describe('SingleSelectListBoxData', () => {
 
         it('throws NO error when adding permitted child using addChild', () => {
           expect(() => {
-            cjsListBox.accessible.addChild(cjsDummy, 0);
+            cjsListBox.accessible.addChild(cjsDummy);
```
Comment (drosenberg123) [inline src/RoleObjects/SingleSelectListBoxData.test.js:105]: - extraneous param

File: src/RoleObjects/RowData.test.js [hunk 5]
```diff
@@ -113,7 +113,7 @@ describe('RowData', () => {
 
         it('throws NO error when adding permitted child using addChild', () => {
           expect(() => {
-            cjsRow.accessible.addChild(cjsDummy, 0);
+            cjsRow.accessible.addChild(cjsDummy);
```
Comment (drosenberg123) [inline src/RoleObjects/RowData.test.js:116]: - extraneous param

File: src/RoleObjects/OrderedListData.test.js [hunk 6]
```diff
@@ -80,7 +80,7 @@ describe('OrderedListData', () => {
 
         it('throws NO error when adding permitted child using addChild', () => {
           expect(() => {
-            cjsOrderedList.accessible.addChild(cjsDummy, 0);
+            cjsOrderedList.accessible.addChild(cjsDummy);
```
Comment (drosenberg123) [inline src/RoleObjects/OrderedListData.test.js:83]: - extraneous param

File: src/RoleObjects/MenuData.test.js [hunk 7]
```diff
@@ -87,5 +89,69 @@ describe('MenuData', () => {
...
+
+          cjsMenuItem2 = new createjs.Shape();
+          AccessibilityModule.register({
+            displayObject: cjsMenuItem2,
+            role: AccessibilityModule.ROLES.MENUITEM,
+          });
+          cjsMenu.accessible.addChild(cjsMenuItem2);
+
+          stage.accessibilityTranslator.update();
+          // console.log(ulEl.outerHTML);
```
Comment (tussharrane) [inline src/RoleObjects/MenuData.test.js:141]: Nit: remove comment

File: jest.config.js [hunk 8]
```diff
@@ -11,10 +11,10 @@ module.exports = {
...
   coverageThreshold: {
     global: {
-      branches: 46,
-      functions: 66,
-      lines: 56,
-      statements: 56,
+      branches: 56,
+      functions: 76,
+      lines: 66,
+      statements: 66,
```
[the remark this answers is not in the document]
Comment (drosenberg123) [reply jest.config.js:17]: didn't even think about it - good catch!

File: src/RoleObjects/AccessibilityObject.test.js [hunk 9]
```diff
@@ -1,43 +1,394 @@
...
+
+    describe('accessible options getters and setters', () => {
+      [
+        { property: 'atomic', ariaAttr: 'aria-atomic', newVal: false },
+        { property: 'busy', ariaAttr: 'aria-busy', newVal: false },
+        {
+          property: 'dropEffects',
+          ariaAttr: 'aria-dropeffect',
+          newVal: 'copy',
+        },
```
[the remark this answers is not in the document]
Comment (drosenberg123) [reply src/RoleObjects/AccessibilityObject.test.js:142]: prettier might override but I'll try

File: src/RoleObjects/MenuData.test.js [hunk 10]
```diff
@@ -87,5 +89,69 @@ describe('MenuData', () => {
         expect(cjsMenu.accessible.enableKeyEvents).toEqual(newVal);
       });
     });
+
+    describe.skip('"onKeyDown" event listener', () => {
```
[the remark this answers is not in the document]
Comment (drosenberg123) [reply src/RoleObjects/MenuData.test.js:93]: nope - good catch!

File: src/RoleObjects/ComboBoxData.test.js [hunk 11]
```diff
@@ -5,6 +5,7 @@ import { parentEl, stage, container } from '../__jestSharedSetup';
 describe('ComboBoxData', () => {
   describe('register role', () => {
     let cjsComboBox;
+    let cjsSpan;
```
Comment (sumaiyakhan2013) [inline src/RoleObjects/ComboBoxData.test.js:8]: so can we combine all the variables under single "let". In all the files below.
`let cjsComboBox, cjsSpan, comboBoxEl, isExpanded, isReadOnly, isRequired, shouldAutoComplete;`

Comment (mmoss) [reply src/RoleObjects/ComboBoxData.test.js:8]: It's easier for diffs to keep them separate.

File: src/RoleObjects/AccessibilityObject.test.js (hunk 9 above)
[the remark this answers is not in the document]
Comment (drosenberg123) [reply src/RoleObjects/AccessibilityObject.test.js:141]: no go, prettier (autoformatting) wins

**Review by mmoss — APPROVED** [review]
Almost up to 80% across the board!

**Pull request merged**